core-ui/src/components/Predefined/Details/CustomResourceDefinitions/CRCreate.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import * as jp from 'jsonpath';
+import { ResourceForm } from 'shared/ResourceForm';
+import { createTemplate } from './templates';
+import LuigiClient from '@luigi-project/client';
+
+const CRCreate = ({ onChange, formElementRef, crd }) => {
+  const namespace =
+    crd.spec.scope === 'Namespaced' && LuigiClient.getContext().namespaceId;
```

Comment:
```suggestion
function CRCreate({ onChange, formElementRef, crd }) {
  const { namespaceId: namespace } = useMicrofrontendContext();
```

Let's keep it consistent.